apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -363,6 +365,7 @@ function DeleteQuestionModal({
   assessmentsWithQuestion: SelectedAssessments[];
   csrfToken: string;
 }) {
+  const plainUrlPrefix = config.urlPrefix;
```

Comment:
I'd be inclined to inline this as `plainUrlPrefix: config.urlPrefix` below.

Reply:
Can do. Do you think the badge should default to `config.urlPrefix` if `plainUrlPrefix` is not provided?

Reply:
Nevermind, better not, otherwise it introduces a dependency on config that may end up in client-side code.